lib/jquery.js

```diff
@@ -0,0 +1,25 @@
+//A hover effect that gives the tile an outline, or give it a box shadow.
+$(".dogs").hover(function(){
+    $(this).addClass("hoverborder");
+}, function() {
+    $(this).removeClass( "hoverborder");
+  }
+);
+
+//Zooms in on the dog photo
+$(".dog-image").hover(function(){
```

Comment:
Nice job with the effects. Would see if you could mess around with z-index in your hoverborder class, so the image is always on top. Sometimes it ends up behind the other dog tiles